### ISSUE TITLE
[Bug]: Ini file not created on install for specific pattern in PHP_INI_SCAN_DIR

### ISSUE THREAD

**Issue by MitchelAnthony** [DataDog/dd-trace-php#2949]
### Bug report

The installer (datadog-setup.php) reads and uses, [via this command](https://github.com/DataDog/dd-trace-php/blob/49b3dc01d5889a21a39ebe8d7149029e25a72397/package/post-install.sh#L192), the additional php ini directory set through `PHP_INI_SCAN_DIR` to create a symlink to the `98-ddtrace.ini` file. However, if the env var starts with [the perfectly legal : (colon) character](https://www.php.net/manual/en/configuration.file.php#configuration.file.scan), the creation of the symlink fails. This is not communicated to the user, however, and the result is that the datadog agent is not loaded.

Example output (final lines only):
```
Created INI file ':/usr/local/etc/php/app.conf.d/98-ddtrace.ini'
Installation to 'php (/usr/local/bin/php)' was successful
```

The workaround we're currently using is to remove the colon from the env var just before running the installer and restoring it after the install, but it should probably be fixed at the source.

### PHP version

8.3.13

### Tracer or profiler version

master

**Comment by bwoebi** [comment]
Hey @MitchelAnthony,

this was fixed in #2941 and will be working fine starting next release!

**Comment by MitchelAnthony** [comment]
I see! Thanks! For some reason it wasn't in the search results when I searched for a similar issue.
